README.md

```diff
@@ -1,2 +1,98 @@
-# hpe-3par-cinder-rhosp16
-This repository contains Dockerfile used for building the custom container for RHOSP16 HPE 3PAR Cinder Driver 
+# HPE 3PAR and Primera Cinder volume custom container for RHOSP16
+
+## Overview
+
+This page provides detailed steps on how to enable the containerization of HPE 3PAR and Primera Cinder driver on top of the OSP Cinder images.
+
+The custom Cinder container image contains following package:
+
+python-3parclient 4.2.11
+
+## Prerequisites
+
+* Red Hat OpenStack Platform 16.
+
+* HPE 3PAR array 3.3.1.
```

Comment:
As part of heading we say 3par and primera, but here we have validated only against 3.3.1.
Are we sure it will work on primera 4.x array too?